apps/prairielearn/src/ee/lib/ai-grading.ts

```diff
@@ -228,6 +308,22 @@ async function ensureSubmissionEmbedding({
   return new_submission_embedding;
 }
 
+function validateRubric(old_rubric_items: RubricItem[], new_rubric_items: RubricItem[]): void {
+  if (old_rubric_items.length !== new_rubric_items.length) {
+    throw new Error('rubric modified between trieval and grade insertion');
+  }
+  for (let i = 0; i < old_rubric_items.length; i++) {
+    const old_item = old_rubric_items[i];
+    const new_item = new_rubric_items[i];
+    if (
+      old_item.description !== new_item.description ||
+      old_item.explanation !== new_item.explanation
+    ) {
+      throw new Error('rubric modified between trieval and grade insertion');
```

Comment:
What is this message supposed to say? I don't think 'trieval' is a word.

Reply:
I might have deleted some characters by mistake - I meant to say rubric retrieval. I'll fix it